examples/tensorflow-mnist-pixel-threshold/task-plugins/securingai_custom/evaluation/pixelthreshold.py

```diff
@@ -14,31 +14,28 @@
 #
 # ACCESS THE FULL CC BY 4.0 LICENSE HERE:
 # https://creativecommons.org/licenses/by/4.0/legalcode
-"""A task plugin module for the Pixel Threshold evasion attack.
 
+"""A task plugin module for the Pixel Threshold evasion attack.
 The Pixel Threshold [kotyan2019]_ is an evasion attack that attempts to
 fool a trained classifier by perturbing a test image within a certain threshold
 of pixels. This task plugin uses the Adversarial Robustness Toolbox's
 [art2019]_ implementation of the Pixel Threshold attack |pt_art|.
-
 References:
     .. [art2019] M.-I. Nicolae et al., "Adversarial Robustness Toolbox v1.0.0,"
         2019. [Online]. Available:
        `arXiv:1807.01069v4 [cs.LG] <http://arxiv.org/abs/1807.01069v4>`_.
```

Comment:
```suggestion
    .. [art2019] M.-I. Nicolae et al., "Adversarial Robustness Toolbox v1.0.0,"
        2019. [Online]. Available:
       `arXiv:1807.01069v4 [cs.LG] <http://arxiv.org/abs/1807.01069v4>`_.

```
Restore removed whitespace to ensure restructured text parser doesn't get confused.